pkg/dependency/parser/python/pip/parse.go

```diff
@@ -53,10 +68,14 @@ func (p *Parser) Parse(r xio.ReadSeekerAt) ([]ftypes.Package, []ftypes.Dependenc
 		line = rStripByKey(line, commentMarker)
 		line = rStripByKey(line, endColon)
 		line = rStripByKey(line, hashMarker)
-		s := strings.Split(line, "==")
+
+		s := p.splitLine(line)
 		if len(s) != 2 {
 			continue
 		}
+		if p.useMinVersion && strings.Index(s[1], ".*") > -1 {
```

Comment:
IIUC we can check `.*` as suffix:
```bash
root@5c86285e4f39:/app# pip install django==5.*.*
ERROR: Invalid requirement: 'django==5.*.*': .* suffix can only be used with `==` or `!=` operators
    django==5.*.*
          ~~~~~~^
root@5c86285e4f39:/app# pip install django==4.*.1
ERROR: Invalid requirement: 'django==4.*.1': Expected end or semicolon (after version specifier)
    django==4.*.1
          ~~~~~^
root@5c86285e4f39:/app# pip install django==5.*
Collecting django==5.*
  Downloading Django-5.1-py3-none-any.whl.metadata (4.2 kB)
```

Reply:
Do we need this check (I mean `strings.Index(s[1], ".*") > -1`)?
It seems like we can always replace the `.*` suffix if `p.useMinVersion==true`.
Or am I missing something?

Reply:
@DmitriyLewen ok, removed this check

Reply:
no, it doesn't work correctly. updating.